libs/sdk-ui-vis-commons/src/utils/calculateCustomHeight.ts

```diff
@@ -13,17 +19,22 @@ export enum HEIGHT {
     LARGE = 104,
 }
 
-export enum FONT_SIZE {
+enum FONT_SIZE {
     SMALLEST = 30,
     SMALL = 36,
     MEDIUM = 46,
     LARGE = 50,
 }
 
+/**
+ * @public
```

Comment:
This should be `@internal` since it is in the `vis-commons` package and only for internal use.